front/app/components/CustomFieldsForm/generateYupSchema.ts

```diff
@@ -96,47 +96,48 @@ const generateYupValidationSchema = ({
             ? formatMessage(messages.titleRequired)
             : formatMessage(messages.descriptionRequired);
 
-        schema[key] = enabled
-          ? validateAtLeastOneLocale(requiredMessage, {
-              validateEachNonEmptyLocale: (schema) => {
-                let fieldSchema = schema;
-
-                if (input_type === 'text_multiloc') {
-                  // Apply standard character limits to text_multiloc
-                  if (min_characters) {
-                    fieldSchema = fieldSchema.min(
+        schema[key] =
```

Comment:
Not sure how the UI works atm but I think if the field is not required, we shouldn't show the option to set character limits. They won't work and it might be confusing.